### PR TITLE
PAPP-21257 App fails periodically due too many active sessions

### PR DESCRIPTION
**Pull request by splunk-jessica**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
- [x] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [x] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
Added active session cleanup

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by rgil-splunk** [comment]
please update `unreleased.md` with relevant release note changes, see https://confluence.splunk.com/display/APD/Splunk-SOAR-Connectors+Overview#SplunkSOARConnectorsOverview-GeneratingReleaseNotes for more details

File: release_notes/unreleased.md [hunk 1]
```diff
@@ -1 +1,7 @@
 **Unreleased**
+* Added the following actions [PAPP-9260, PAPP-9261, PAPP-21056, PAPP-21057]
+  * add host
+  * delete host
+  * list hosts
+  * logout session
```
Comment (rgil-splunk) [inline release_notes/unreleased.md:6]: I'm guessing this was from a prior PR and the changes haven't been released yet?

**Review by rgil-splunk — APPROVED** [review]

File: release_notes/unreleased.md (hunk 1 above)
Comment (splunk-jessica) [reply release_notes/unreleased.md:6]: correct

**Review by tonyc-phantom — CHANGES_REQUESTED** [review]
I think you forgot to disable this after you finished debugging :)

File: checkpoint_connector.py [hunk 2]
```diff
@@ -658,8 +654,9 @@ def handle_action(self, param):
 if __name__ == '__main__':
 
     import sys
-    # import pudb
-    # pudb.set_trace()
+
+    import pudb
+    pudb.set_trace()
```
Comment (tonyc-phantom) [inline checkpoint_connector.py:659]: I think you forgot to disable this again after debugging :)

**Review by tonyc-phantom — CHANGES_REQUESTED** [review]

**Review by tonyc-phantom — APPROVED** [review]
Looks good!

**Review by rgil-splunk — APPROVED** [review]

**Pull request merged**